<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Resource">
     <attribute name="title">
      <string>资源</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_cpu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>CPU</string>
      </property>
      <widget class="QProgressBar" name="progressBar_cpu">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU </string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>461</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>内存和交换区</string>
      </property>
      <widget class="QProgressBar" name="progressBar_Mem">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_Swap">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Memory</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Swap</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Used:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mUsed">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mTotal">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Used:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>140</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sUsed">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_sTotal">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>140</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Process">
     <attribute name="title">
      <string>进程</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_kill">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>470</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_proc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ModuleInfo">
     <attribute name="title">
      <string>模块信息</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_remove">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>470</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_module">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SysInfo">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>处理器信息</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>处理器:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cpuname">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cpu core:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cpucore">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>cache size:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cachesize">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>操作系统信息</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_sysinfo">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_xuanxiang">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionShutdown"/>
    <addaction name="actionReboot"/>
   </widget>
   <widget class="QMenu" name="menu_wenjian">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_newproc"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_chakan">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action_refresh"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_wenjian"/>
   <addaction name="menu_xuanxiang"/>
   <addaction name="menu_chakan"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShutdown">
   <property name="text">
    <string>关机</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>重启</string>
   </property>
  </action>
  <action name="action_newproc">
   <property name="text">
    <string>运行新任务</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>立即刷新</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
